tests/console/postgresql_server.pm

```diff
@@ -27,8 +27,8 @@ sub run {
     # start the postgresql service
     assert_script_run 'systemctl start postgresql.service', 200;
 
-    # check the status
-    assert_script_run 'systemctl show -p ActiveState postgresql.service | grep ActiveState=active';
+    # check the status, if failed, get possible cause
+    assert_script_run 'systemctl show -p ActiveState postgresql.service | grep ActiveState=active || systemctl status postgresql.service';
```

Comment:
assert_script_run verifies return value of last command
 - before it was checking "grep ActiveState=active"
 - with your change it will check "systemctl status postgresql.service" - so it will never fail

That means you are disabling the check!

Reply:
I did some tests stopping postgresql just before that line and it worked fine. I guess 
(systemctl show -p ActiveState postgresql.service | grep ActiveState=active) || systemctl status postgresql.service will restore original behaviour.

Reply:
Brackets won't change the behavior - you can replace this line with
assert_script_run 'systemctl status postgresql' - it will have the same effect.

It works because of "systemctl status" return value - https://github.com/systemd/systemd/issues/1092